testutil/src/lib.rs

```diff
@@ -40,10 +40,10 @@ impl PinnedVersions {
             .ok_or("failed to find msrv: no `rust-version` key present")?
             .to_string();
```

Comment:
We'll need to find a way to work around https://github.com/oli-obk/cargo_metadata/issues/253.